### DISCUSSION TITLE
Snackbar in using statement

### DISCUSSION THREAD

**Discussion by NXDMN** [CommunityToolkit/Maui#1141] [Q&A] [answered]
Hi I would like to use Snackbar in using statement like 
```(using var snackbar = await SnackbarHelper.Loading("Some text"))```
in which the Loading method i created with reference to the one in Xamarin,  https://github.com/Baseflow/XF-Material-Library/blob/fef56fb66de1ff7fc198745f200883f6ac489c20/XF.Material/UI/Dialogs/MaterialSnackbar.xaml.cs#L49, and then I would like to show multiple snackbars one by one in order, but with different Message. However, there is no Text setter so I cannot do like ```snackbar.Text = "new message"```
does it mean i can only create my own Snackbar or call Make and Show multiple times?

**Comment by VladislavAntonyuk** [comment, marked as answer]
Snackbar is immutable so you need to call it multiple times.

**Comment by CliffAgius** [comment]
Closed as answered
